website/docs/r/emr_cluster.html.markdown

```diff
@@ -45,6 +45,7 @@ EOF
 
   master_instance_group {
     instance_type = "m4.large"
+    market        = "SPOT"
```

Comment:
Nit: I'm not sure our example configuration should recommend `SPOT` for the master instance group, since presumably it can get pre-empted.